pkg/restutil/restutil.go

```diff
@@ -73,17 +70,17 @@ func validateURL(url string) error {
 }
 
 func getResponse(httpMethod, url string, payload interface{}, auth *Auth, headers map[string]string) ([]byte, int, error) {
-	log = logInstance.GetLogInstance()
+
 	var err error
 	err = validateURL(url)
 	if err != nil {
 		return nil, 0, err
 	}
 
-	log.Tracef("%s: %s", httpMethod, url)
+	logrus.Debugf("%s: %s", httpMethod, url)
```

Comment:
Maybe add some comments about why we use logrus directly in this file.

Reply:
This is done to avoid cyclic dependency, added comment.